tests/apis/platformvm/api.test.ts

```diff
@@ -1608,7 +1608,8 @@ describe("PlatformVMAPI", (): void => {
 
       serialzeit(tx1, "AddDelegatorTx")
     })
-
+    /*
+   TODO: @VjeraTurk Change to buildAddCaminoValidatorTX
```

Comment:
Yes, good catch. This will be done later based on https://github.com/chain4travel/caminojs/blob/VjeraTurk/add-vote-example/examples/platformvm/buildCaminoAddValidatorTx.ts